Calibration/HcalAlCaRecoProducers/test/AlCaHBHEMuonProducerFilter_cfg.py

```diff
@@ -0,0 +1,52 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run2_2018_cff import Run2_2018
+process = cms.Process("AlCaHBHEMuon",Run2_2018)
+
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load("Configuration.StandardSequences.GeometryRecoDB_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+process.load('Configuration.StandardSequences.AlCaRecoStreams_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+from Configuration.AlCa.autoCond import autoCond
+process.GlobalTag.globaltag=autoCond['run2_mc']
```

Comment:
`run2_mc` is for 2016 MC, is this intended?

Reply:
the rest of the config is for HLTPhysics data, right?

Reply:
Corrected. Here we do this for HLTPhysics - but it is just an example